docusaurus/docs/Android/04-compose/04-message-components/02-message-list-header.mdx

```diff
@@ -56,30 +63,24 @@ fun MessageListHeader(
 )
 ```
 
-* `onBackPressed`: Handler for the `BackButton` click action.
+* `onBackPressed`: Handler for the header `BackButton` click action.
 * `onHeaderActionClick`: Handler when the user clicks on the title for more information about the channel.
 
+:::note
+The `leadingContent` uses the `onBackPressed` parameter as its default action, while the `centerContent` uses the `onHeaderActionClick` parameter as its default action. If you want to keep the same UI but change the behavior, you need to override `onBackPressed` or `onHeaderActionClick`. Otherwise, skip over to [Customization](#customization) to learn how to customize the UI.
+:::
+
 To customize these actions, simply use the `MessageListHeader` with the rest of your UI components, like within `setContent`, and pass in the required actions:
 
 ```kotlin
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        // Load required data
-
-        setContent {
-            Column(Modifier.fillMaxSize()) {
-                MessageListHeader(
-                    ..., // State
-                    onBackPressed = { finish() },
-                    onHeaderActionClick = { 
-                      // Show your custom UI
-                    }
-                )
-
-                // Rest of your UI
-            }
-        }
+MessageListHeader(
+    ..., // State
+    onBackPressed = { finish() },
+    onHeaderActionClick = {
+      // Show your custom UI
     }
+)
+}
```

Comment:
I think this should just be:

```
MessageListHeader(
    ..., // State
    onBackPressed = { finish() },
    onHeaderActionClick = {
      // Show your custom UI
    }
)
```

Reply:
Presumably you mean remove the left over `}`